### ISSUE TITLE
read.ctd() can fail on computing sigmaTheta

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#2161]
I don't think it should be computing sigmaTheta anyway.  We have accessors that will let the user get that quantity if they want, or sigma0 for a gsw version, etc.

It's weird because if I do `[["longitude"]]`, I see that the object has a longitude.  And the file has it:
```
* NMEA Latitude = 47 50.097 N
* NMEA Longitude = 069 48.960 W
```

I don't remember seeing this with other files, but I would be surprised if the cnv format has changed in 15 years.

I'll look into this later today.

```R
> file
[1] "~/data/archive/sleiwex/2008/fielddata/2008-06-26/CoriolisII/seabirdCTD/cnvfiles/2008-06-26_st070.cnv"
> options(oceEOS="gsw")
> read.ctd(files)
Error in swSigmaTheta(median(res[["salinity"]]), median(res[["temperature"]]),  : 
  must supply longitude
> traceback()
4: stop("must supply longitude") at sw.R#1960
3: swSigmaTheta(median(res[["salinity"]]), median(res[["temperature"]]), 
       0) at ctd.sbe.R#1294
2: read.ctd.sbe(file, encoding = encoding, columns = columns, station = station, 
       missingValue = missingValue, deploymentType = deploymentType, 
       monitor = monitor, debug = debug, processingLog = processingLog, 
       ...) at ctd.R#4181
1: read.ctd(files[1])
```

**Comment by dankelley** [comment]
Oh, it's doing that so it can compute pressure.  These files have "depSM" (depth in salt water, in metres) but not pressure, so the code is computing pressure, and we need density for that.  I think I ought to call the unesco version to get density ... after all, gsw is set up to agree with unesco on density.  (Or almost exactly agree ... who cares since  we are likely beyond the ability to discern differences, with this measurement method.)

**Comment by dankelley** [comment]
Addressed in "develop" commit fd6488ec785c733f3036f0412b0f33c2b8bd6a37.

I will check in an hour to see if this builds and checks ok locally and remotely, and will reopen this if there is a problem.
